### ISSUE TITLE
Error in netVisual_embeddingPairwise()

### ISSUE THREAD

**Issue by wajm** [sqjin/CellChat#498]
Thanks for great tool!

I tried to analyze with netVisual_embeddingPairwise function with 2 groups.
but I got error message
>netVisual_embeddingPairwise(cellchat, type = "functional", label.size = 3.5)
2D visualization of signaling networks from datasets 1 2 
Error in data.frame(x = Y[, 1], y = Y[, 2], Commun.Prob. = prob_sum/max(prob_sum),  : 
  arguments imply differing number of rows: 76, 78

I checked my data in cellchat@netP$similarity$functional
cellchat@netP$similarity$functional$matrix  -> 1-2 double [78 x 78]
cellchat@netP$similarity$functional$dr         -> 1-2 double [76 x 2]
cellchat@netP$similarity$functional$group  -> 1-2 integer [76]

I think that 2 rows:76, 78 lost in netEmbedding.

Is there any solution for this?

**Comment by sqjin** [comment]
@wajm Is it possible to share me an cellchat object that can replicate your error?

**Comment by wajm** [comment]
> @wajm Is it possible to share me an cellchat object that can replicate your error?

Of course, let me know your private mail. I will share via Gdrive link.

**Comment by wajm** [comment]
Okay, It is working nicely. Thanks a lot.
